docs/en_US/reference/experiment_config.rst

```diff
@@ -370,41 +381,42 @@ See algorithm's document for supported value.
 TrainingServiceConfig
 ^^^^^^^^^^^^^^^^^^^^^
 
-One of following:
+One of the following:
 
 - `LocalConfig`_
 - `RemoteConfig`_
-- `OpenpaiConfig <openpai-class>`_
+- :ref:`OpenpaiConfig <openpai-class>`
```

Comment:
why OpenpaiConfig is different?

Reply:
see https://github.com/microsoft/nni/pull/3711#discussion_r644574928